scripts/circle-ci-tests.sh

```diff
@@ -57,7 +57,7 @@ else
             echo "Finding pylint violations and storing in report..."
             # HACK: we need to print something to the console, otherwise circleci
             # fails and aborts the job because nothing is displayed for > 10 minutes.
-            paver run_pylint -l $PYLINT_THRESHOLD | tee pylint.log || EXIT=1
+            paver run_pylint -l $LOWER_PYLINT_THRESHOLD -u $UPPER_PYLINT_THRESHOLD | tee pylint.log || EXIT=1
```

Comment:
Looks like this and generic-ci-tests.sh need to be updated to the colon-range.

Reply:
Whoops - forgot that...

Reply:
Fixed now.